src/docfx/restore/RestoreFile.cs

```diff
@@ -44,15 +44,10 @@ public static async Task Restore(string url, Config config)
 
             using (InterProcessMutex.Create(filePath))
             {
-                if (!File.Exists(filePath))
-                {
-                    PathUtility.CreateDirectoryFromFilePath(filePath);
-                    File.Move(tempFile, filePath);
-                }
-                else
-                {
-                    File.Delete(tempFile);
-                }
+                PathUtility.CreateDirectoryFromFilePath(filePath);
+
+                File.Copy(tempFile, filePath, true);
```

Comment:
Prefer `File.Move` over `File.Copy` here to prevent data corruption. `File.Move` is mostly an atomic operation while `File.Copy` isn't. When docfx crashed in the middle of `File.Copy` for a large files, the target file exist but only contains partial data.